plugins/modules/system/java_keystore.py

```diff
@@ -97,6 +97,24 @@
       - openssl
       - cryptography
     version_added: 3.1.0
+  keystore_type:
+    description:
+      - Type of the Java keystore.
+      - When this option is omitted and the keystore doesn't already exist, the
+        behavior is to follow C(keytool) default store type, that depends on
+        Java version, and is C(pkcs12) since Java 9, and C(jks) before (may also
+        be C(pkcs12) if new default has been backported to this version).
```

Comment:
```suggestion
      - When this option is omitted and the keystore doesn't already exist, the
        behavior follows C(keytool)'s default store type which depends on
        Java version; C(pkcs12) since Java 9 and C(jks) prior (may also
        be C(pkcs12) if new default has been backported to this version).
```